### PR TITLE
Added blog for poster presentation

### PR DESCRIPTION
**Pull request by suprith4989**
This blog also contains inputs from Devyani Kota.

### CODE REVIEW

**Comment by kracekumar** [comment]
Can you remove your name and email ? We would like to use markdown, can you make the changes ?
@harisibrahimkv can review the content.

**Comment by harisibrahimkv** [comment]
Content looks good. I don't know about the Poster session to nitpick on
anything. :)

On 28 August 2015 at 15:10, Kracekumar Ramaraju <notifications@github.com>
wrote:

> Can you remove your name and email ? We would like to use markdown, can
> you make the changes ?
> @harisibrahimkv <https://github.com/harisibrahimkv> can review the
> content.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonindia/inpycon-blog/pull/24#issuecomment-135718584>
> .
>



-- 
Haris Ibrahim K. V.
http://sosaysharis.wordpress.com
@harisibrahimkv <http://twitter.com/harisibrahimkv>

**Comment by suprith4989** [comment]
@harisibrahimkv Please check the MoM sent by Chandan Kumar at Aug 20

**Comment by suprith4989** [comment]
@kracekumar Sure

File: content/blog/poster_session_2015.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
+======================================
+Poster Presentation - Pycon India'2015
+======================================
+
+What is Poster Presentation?
+============================
+Ideally, Poster presentation/session is an informal way to present your ideas, or research information at the conference with an academic focus and reach out to a larger audience on your topic. Basically the poster presentations are different from the soft, projector based presentations, where the  presenter/individual/speaker accompanies a poster, posts it on a black/white board in the space provided, stand there, talk to the rolling audience, explain the concept, take maximum questions on his topic. Each presentation is usually presented over a period of 10 minutes or more.
```
Comment (kracekumar) [inline content/blog/poster_session_2015.rst]: Ideally isn't required here.

File: content/blog/poster_session_2015.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+====================
+1. Professionals
+2. Lecturers
+3. College Students
+4. Speakers 
+
+How to submit Posters?
+======================
+Through Junction:
+https://in.pycon.org/cfp/pycon-india-2015/proposals/
```
Comment (kracekumar) [inline content/blog/poster_session_2015.rst]: We will change this url, I will create one and update the link here.

Comment (kracekumar) [inline content/blog/poster_session_2015.rst]: https://in.pycon.org/cfp/pycon-india-2015-poster-session/proposals/.

**Comment by vnbang2003** [comment]
@suprith4989 Are we ready with it ?

**Comment by suprith4989** [comment]
@kracekumar @vnbang2003 @harisibrahimkv I've made the changes as suggested above. And also do I need to remove 'TBD' in the Posters Selection section? Please let me know.

**Comment by kracekumar** [comment]
@suprith4989 Can you change the `rst` to `markdown` ? Yes you can remove `TBD`.

**Pull request merged**